libraries/RC_Channel/RC_Channel.h

```diff
@@ -601,6 +608,13 @@ class RC_Channels {
 
     // true if GCS is performing a RC calibration
     bool gcs_is_calibrating;
+
+    // clear overrides by radio input
+    void clear_overrides_by_rc_in();
+
+    bool clear_overrides_by_rc(void) const {
```

Comment:
my guess is that other devs may say we don't need this small function that is only called from one place.  Also its name is so close to the method above that it is easy to get confused about which is which.  I'd say we should remove this one but I don't feel strongly about it.

Reply:
I agree with you.
In fact, I had trouble naming the function.